### ISSUE TITLE
Fix inconsistency of on/off and status.

### ISSUE THREAD

**Issue by wasndas** [sdss/lvmnps#33]
The commands on/off are inconsistent, for on/off its:

```
@parser.command()
@click.argument("OUTLET", type=str)
@click.argument("PORTNUM", type=int, required=False)
@click.option(
    "--switch",
    type=str,
    help="Address this switch specifically. Otherwise the first switch "
    "with an outlet that matches NAME will be commanded.",
)
```

and for status its:

```
@parser.command()
@click.argument("SWITCHNAME", type=str, required=False)
@click.argument("PORTNUM", type=int, required=False)
@click.option(
    "-o",
    "--outlet",
    type=str,
    help="Print only the information for this outlet.",
)
```

eg:

`lvmnps on skyw.pwi`

and for status its:

`lvmnps status -o  skyw.pwi`

**Comment by albireox** [comment]
I agree there's some inconsistency but I think it's for a good cause. For status I think it's more likely that one wants to get the status of all the outlets in a switch instead of an outlet. For on/off one always wants to address an outlet (I think it's very unlikely one will want to turn on/off an entire switch.

In my opinion this inconsistency is ok, but if we wanted to consolidate I would vote for keeping `lvmnps on/off` as it is and change `lvmnps status` to address an outlet by default (and change the `-o` flag with a `-s` for switch).

**Comment by wasndas** [comment]
Still, its confusing.

In the original implementation from one year ago, there was no -s and -o at all.
As it is still described in the README you could use on/off/status:

 1. with the unique switch name and optional a port number.
 2. with the unique port name - which should be the preferred usage.

Both switch names and port names should be unique, lvmnps took care internally if a switch (optional with port number) or a named port was addressed.

I my opinion the normal usage with on/off/status should be via unique port names, maybe multiple port names as parameters would be even better.

If a switch (and optional with a port number) has to be addressed then an option -s should be used or just revert the interface as it was.
